src/language-html/syntax-vue.js

```diff
@@ -50,14 +55,27 @@ function parseVueFor(value) {
     if (iteratorMatch[2]) {
       res.iterator2 = iteratorMatch[2].trim();
     }
+
+    if (
+      (res.iterator2 && (!res.iterator1 || !res.alias)) ||
+      (res.iterator1 && !res.alias)
+    ) {
+      return;
+    }
   } else {
     res.alias = alias;
   }
 
+  const left = `${[res.alias, res.iterator1, res.iterator2]
+    .filter(Boolean)
+    .join(",")}`;
```

Comment:
Why this write in template original..

Reply:
Yeah, it makes sense to remove this template

Reply:
actually i found another bug

Reply:
it should not `.filter(Boolean)` the statements.
suppose:

```html
<div id="hello-vue" class="demo">
  <div v-for="(,a,b) of 'abcd'">
    {{a}}, {{b}}
  </div>
</div>
```
should print 
```
0,
1,
2,
3,
```
but now the if i call `parseVueFor('(,a,b) of "abcd"')` it returns undefined

Reply:
Moreover, if you run this sample `v-for="(,a,b) of 'abcd'"` in prettier@2.5.1 
[link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeeBbADgGwIbwB8AOlAARmoAmAlgG5l0C0AZhAE4C8xIAFADR5+AIwCUZCCzJgehVAHpadElAWZcBOCpD8QELDBrQAzslB527CAHcAChYSmUeHNbwBPU7uHs8YANZwMADKeBhwADI0UHDILC7GcN6+AUHBWH7RAObIMOwArkkgcBjCcFRU5RF4UFn5eFlwAGIcGASGtcggePkwEDogABYwGDgA6oM08MYZYHDBjlP0U+5dYMZeINGJ7DC2vlltcQlFAFbGAB7B2ThwAIr5EPDHOIm6Gew7XXSFA1js0RgYxoVBgg2QAA4AAzvKyJMa+LBdf5wHZ0WK6ACOj3g+30Tm6xiYMXK5QG7Dg2JoFP2DSOSHiryKiQwNFyBWZN3uONiDJOuhgeGEwNB4KQACYBb4aDhsgBhCAYenFYwAVgG+USABUhU5GW8QD84ABJKCVWDBMAAgwAQTNwRg7luL0SAF9XUA)
it removed the first comma, which is incorect

Reply:
do you guys think this worth another issue?

Reply:
I've posted this new issue here: https://github.com/prettier/prettier/issues/12114

Reply:
I've fixed this case here.

Reply:
Added a test for your case https://github.com/prettier/prettier/pull/12113/commits/6bbf42bb043622b0e30b68c342dec0c5ae01acb6